### PR TITLE
Revert "Hooch no longer heals assistants."

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Reverts tgstation/tgstation#64693

Don't remove the trait, its unrelated.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
the trait was literally exclusively used by hooch mso

**Comment by Rohesie** [comment]
Unrelated to?

**Comment by optimumtact** [comment]
I'll be handling this PR

**Comment by SinfulBliss** [comment]
EXTREMELY based MSO

**Comment by Iamgoofball** [comment]
he weekbanned me from the discord for suggesting he make a revert PR to just revert the trait instead of reverting the hooch nerf lma
ShareX_BT7y1DjjqE
o

**Comment by optimumtact** [comment]
I don't know exactly what isn't clear about I'll handle this, but that aint it chief.

**Comment by optimumtact** [comment]
My understanding of the concern here is the following, based on discussions I've had with MSO,  @MrStonedOne please feel free to correct + clarify any further.

One) MSO in general dislikes innate job related traits - such as in this example, assistants getting healed for hooch. We try as hard as possible to have IC explanations for much of what is going on and this violates that. He also views, and I tend to agree, that the world of job's belongs to the Head of Personnel and that having such innate traits waters down/ prevents the HoP properly changing peoples jobs. This is a very "soul" argument, but I think it's a valid viewpoint to have and one I also tend to agree with.

Two) The PR did not attempt to address the underlying problem of all innate job related healing drinks/items, but instead was relating only to one existing one, so rather than fixing a problem it was only covering up one of the more obvious examples.

Three) The PR was not well justified, and is clearly influenced by recent events in game and on the forums. While rage/ided pr's are a somewhat valid form of expression, it's our duty as maintainers to ensure that the PR addresses an underlying problem, and isn't solely as a result of some in game issue.

I am still deciding on what to do, but there are some things informing my general viewpoint that early on have me leaning towards removing job related drink effects as a general thing.

One) I agree in general with MSO's views on the HoP and my view is that we should avoid as much as possible any kind of innate traits relating to jobs.

Two) The general maintainer consensus is that innate job related traits are bad, and there are a number of existing frameworks that help manage that -  Quirks and also Skillchips

Three) I'm not convinced that the healing drinks achieve their intended goal, which was encouraging people to come to the bar and engage with the bartender, I do not have good answers to how we can drive more player engagement with the bar, and the food/service jobs, but I do not think that this approach is working.


I am going to open the PR comments up for general discussion but I do not want to see any comments on discord, or the recent ban appeal that most likely lead to the original PR.

If people can discuss the above three points instead I would be greatly obliged.


In terms of replacement, I forsee a path where there is a drunken fighter quirk, or similar, that gives the user some small passive healing from drinks, but in return while they're not actively drunk, they have a lower stamina cap and slower stamina regen.

This isn't a job related trait, so sidesteps most of the issues above, and is a more interesting choice because it presents the user with some drawbacks.

**Comment by alphanerdd** [comment]
I agree with removing most job related quirks, including the healing drinks. It makes job transfers pretty weird, and doesn't really match up with most of the game mechanics. However, if job traits are going to be removed, it needs to be all at once. Removing them piecemeal, such as removing hooch healing, feels a lot more like targeted role nerfs and is just going to create bad community reactions.

The one circumstance I think job quirks are a good thing is with mimes, mainly because ~~I'm biased~~ mimes already have lots of mechanics innately tied to their job, stuff like invisible walls, vows of silence, and the bottle of nothing. The traits are a pretty big part of the job, and removing them would be overall a bad thing.

**Comment by EOBGames** [comment]
> Three) I'm not convinced that the healing drinks achieve their intended goal, which was encouraging people to come to the bar and engage with the bartender, I do not have good answers to how we can drive more player engagement with the bar, and the food/service jobs, but I do not think that this approach is working.

In the case of most of these drinks, this is true- although I would note that Quadruple Sec is something that seccies quite frequently come by and specifically request, as was Screwdriver for engineers (at least prior to rad removal, I can't comment on how it's been since). My guess is that that's down to availability (both of those drinks can be made entirely from the drinks dispensers) and community knowledge (both drinks are more well known than things like Doctor's Delight). I'm not sure if this is easily solvable, but it might be something to consider.

**Comment by optimumtact** [comment]
>  However, if job traits are going to be removed, it needs to be all at once. Removing them piecemeal, such as removing hooch healing, feels a lot more like targeted role nerfs and is just going to create bad community reactions.

This isn't possible for the exact same reason it takes us 5 years to overhaul the sprite direction of the game, we do not have resources we can assign to specific jobs due to working in a volunteer context. If we obeyed this logic and only allowed changes that did everything at once nearly nothing would get done at all.

As for the mime issue, the mime spells come from their book of mimery, so isn't tied directly to the job, the vow of silence and bottle of nothign can be addressed via skillchips

**Comment by Farquaar** [comment]
I like clowns being clumsy. Would there be any plans for that to change? Or would it be treated like a roundstart quirk?

**Comment by optimumtact** [comment]
The mime and clown have had specific role related traits for so long it's unlikely they would change, and I imagine mso is fine with that.

We might look to make them based on aspects of their equipment or skill chips, but we're not going to remove their specific job related content in a hurry.

**Comment by tralezab** [comment]
Bringing this from discord, to source my thoughts on One): I gave myself a challenge when making thieves to think of: non replenishable, non steal target items that the station can reasonably run out of (so a group of people use them, but they cannot be obtained again)

As it turns out, that is nearly nothing. Everything can be ordered or printed if it isn't one of the few unique steal objectives, and even some steal objectives are nonunique like stealing plasma. So if people can always obtain items they need themselves, it seems incredibly hard to enforce someone's worth based on what they hold on themselves. The other problem is that people can hold a lot on themselves. Our current inventory system allows a lot of space for items if you're lazy, and a ludicrous amount of items if you put effort into maximizing space. So if people can hold crazy amounts of items, and almost zero items in the game aren't obtainable solo, it's easy to see why we have aged players bypassing interaction to loot for themselves and enforced rules (on mrp) to stay in your lane.

**Comment by Serxule** [comment]
> Three) The PR was not well justified, and is clearly influenced by recent events in game and on the forums. While rage/ided pr's are a somewhat valid form of expression, it's our duty as maintainers to ensure that the PR addresses an underlying problem, and isn't solely as a result of some in game issue

yeah goof has been making a lot of ideds lately, i dont know whats been going on with him

**Comment by SkeletalElite** [comment]
> Three) I'm not convinced that the healing drinks achieve their intended goal, which was encouraging people to come to the bar and engage with the bartender, I do not have good answers to how we can drive more player engagement with the bar, and the food/service jobs, but I do not think that this approach is working.

Obviously I can't speak for anyone else's experience rather than my own, but when I'm playing a job with a healing drink, I do like to stop by the bar and ask them for a bottle of it to keep on me. Perhaps this happens due to a lack of knowledge that some jobs get healed by drinks?

**Comment by Iamgoofball** [comment]
>yeah goof has been making a lot of ideds lately, i dont know whats been going on with him

i started playing the game again

**Comment by ATH1909** [comment]
> Two) The general maintainer consensus is that innate job related traits are bad, and there are a number of existing frameworks that help manage that - Quirks and also Skillchips

Technically, this is a trait on livers, not minds, ain't it? So you could get clown metabolism by swapping livers with a clown, for example. I guess moving hooch healing (et. al) to skillchips would be an okay way to resolve this? It's not as funny as it being liver-based, though, and it'd remove the ability of chaplains to discern someone's job by reading their liver, which I thought was a neat feature.

> In terms of replacement, I forsee a path where there is a drunken fighter quirk, or similar, that gives the user some small passive healing from drinks, but in return while they're not actively drunk, they have a lower stamina cap and slower stamina regen.

Ain't that just the drunken healing quirk that we already have?

**Comment by ATH1909** [comment]
> As for the mime issue, the mime spells come from their book of mimery, so isn't tied directly to the job, the vow of silence and bottle of nothign can be addressed via skillchips, (Another option for hte bottle is a specific "Mime" type liver organ)

or we could make learning a mime spell give you the ability to declare a vow of silence, like what I tried to do back in https://github.com/tgstation/tgstation/pull/51916. making it tied to a skillchip probably works as well, although that'd make it harder for slimeperson mimes to keep silent in their new bodies, no?

**Comment by castawaynont** [comment]
removing all departmental healing drinks is good. Quad sec still heals for way too much over time, even with its nerf

**Comment by D4C-420** [comment]
2 things

One a terrible choice but one which technically fixes the issue is to give ALL jobs both a unique liver and job specific drink. Its an even playing field then but again this is honestly a bad idea but I wanted to throw it out there because it might drive up bar interactions because I personally do order screw drivers from the bartender when there is one and I have seen sec frequently ask for qaud sec( but that's just my experience on one server). 

Two is that I like the drinks healing being tied to livers more than skill chips because:
 A) if you want to change your job and benefit from a job specific drink it makes you interact with med bay beyond just being healed and revived which are interactions that are kind of lacking beyond implants that most people go to robo for anyway. Just using a skill chip would entail someone getting one then going to library to wait and activate it in themselves. Its a less involved process and from having done it before honestly kind of boring.
 B) Also currently the framework for making new skill chips is non existent and all of them are found in round start locations. Adding this framework brings up issues like printing say more engi chips  (or having some other  way to obtain more . Mind you looting the CE locker is the only non-murder some guy and cut out their brain for one way to gain a chip) which automatically tell you wires or say the chefs CQC which is of course CQC.  Compare that to livers which are organs. Unique organs (like plasma man ones) can already be found in the limb grower so adding a technode for making job specific livers would not be too far fetched. Healing from drinks is a side benefit to whatever job your doing and if your switching to a job from a separate one I do think the specific aspect should make you take extra steps to benefit from it.

**Comment by Maurukas** [comment]
I do not think that healing food and drinks are a good thing - not Hooch, Doctor's Delight, Quad Sec or any of the others. I think they should all go, and that we should have a better system of satiation, weight and hydration. They absolutely do not accomplish their goal of driving interaction with the service department. 

The current hunger system has nothing to it. You're either fat and slow, or skinny and slow. The morale system, which also ties to food and drink is also similarly weak - you're either happy, or you're sad and slow. I think it needs more detail, granularity and more interaction with other systems. Failure chances at certain interactions when you're so hungry you can't think straight? Likelihood of bleeding or breaking a bone when undernourished and injured? 

What about a per character drink preference, that you get a stronger, longer lasting morale boost from? Nothing combat related or healing.

**Comment by carshalash** [comment]
In my 727 hours of playing bartender, I have not once seen an assistant try to power game hooch to heal themselves.

**Comment by Mothblocks** [comment]
That's because the hooch is in maintenance from what I understand

**Comment by carshalash** [comment]
> That's because the hooch is in maintenance from what I understand

2:1 ethanol and welding fuel with enzyme as a reacting agent. The only time I've ever seen people order it they quickly follow up with saying it's for bacchus and requesting the other parts get made.

**Comment by ATH1909** [comment]
> That's because the hooch is in maintenance from what I understand

from what I understand, the primary reason was because hooch was shit- hooch has **100 boozepwr**, which is not insignificant, especially if you have light drinker, which is otherwise one of the best quirks in the game. for reference, screwdriver has 55 boozepwr, standard space beer has 25, and bacchus's blessing has 300. before goof's PR, it healed brute damage at 1/3 of the rate of libital and 1/2 the rate of aiuri- why wouldn't you just use actual healing chems or a white kit instead of a chem that actively impairs you, is addictive, can't heal most of your friends, and risks causing a hardstun (vomiting) if you drink too much of it?

tldr; hooch healing assistants was a flavor feature, not a balance one, which is why I'm sad to see it go even though I don't play assistant (or, uh, anything anymore, I guess). people (who knew what they were doing) were not playing assistant just for the hooch healing, as rohesie implied in the statement they merged the pr with.

**Comment by ATH1909** [comment]
if you think hooch appearing in maint was the problem, though, we can bring back https://github.com/tgstation/tgstation/pull/64741, which was only closed because goof's pr was merged.

**Comment by AlinaStarkova** [comment]
> > As for the mime issue, the mime spells come from their book of mimery, so isn't tied directly to the job, the vow of silence and bottle of nothign can be addressed via skillchips, (Another option for hte bottle is a specific "Mime" type liver organ)
> 
> or we could make learning a mime spell give you the ability to declare a vow of silence, like what I tried to do back in #51916. making it tied to a skillchip probably works as well, although that'd make it harder for slimeperson mimes to keep silent in their new bodies, no?

dont slime clones keep their genetic mutations when they split? would it be a bad thing to let them copy over skillchips too? might make it a way to farm unlimited skillchips but that could be solved via cloned skillchips always breaking when removed no?

as for mime, most of their spells are in a book, sadly said book cant be reproduced in game so you cant make more mimes that way, perhaps let cargo be able to buy mime or clown books, or rather, clown and mimes be able to order them via the order console. vows of silence can come with the book. the worry there is they can order a bunch and just hand them out to the crew, but we could make it so you need the actual mime/clown job to use the book, so long as hop has a way of actually changing peoples jobs in their mob vars such as someones job in the manifest being modified will modify their mobs job trait as well

**Comment by optimumtact** [comment]
I think the approach i want to take is

A) remove job related healing drinks

B) update drunken resilience to give you healing when drunk, but reduce your stamina cap and regen when not drunk
